cmd/auth-rest/startcmd/start.go

```diff
@@ -44,6 +45,27 @@ const (
 	logLevelPrefixFlagUsage = "Default logging level to set. Supported options: CRITICAL, ERROR, WARNING, INFO, DEBUG." +
 		`Defaults to info if not set. Setting to debug may adversely impact performance. Alternatively, this can be ` +
 		"set with the following environment variable: " + logLevelEnvKey
+
+	// OIDC flags
+	oidcProviderURLFlagName  = "oidc-opurl"
```

Comment:
Shouldn't we have some configuration API to setup OIDC providers as we would have multiple providers supported by hub-auth.